Bottom: 38129e3e1131acb31c8e2b99d188386d878e363f
Top:    7299bbb935b6cf18f193dd1f94152894b929ad5a
Author: scott snyder <sss@karma>
Date:   2020-01-31 19:00:04 -0500

Refresh of gitignore

---

diff --git a/.gitignore b/.gitignore
index a39f1afd..42f304ef 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+*~
 config/lcio.properties
 doc/lcio.xml
 doc/index.html
